### ISSUE TITLE
add tool for generating od matrix from route input and taz file

### ISSUE THREAD

**Issue by namdre** [eclipse/sumo#5026]

**Comment by palvarezlopez** [comment]
@namdre this could be usefully in NETEDIT?

**Comment by namdre** [comment]
@palvarezlopez Yes. But I would not add special functions. Instead I would solve this via #4138 eventually.

**Comment by namdre** [comment]
Note that the tools/output/vehroute2amitranOD.py is very similar in functionality. However it assumes that each vehicle in the output has attributes `fromTaz` and `toTaz` set.

The new tool should be able to load a route file and TAZ file instead.

**Comment by namdre** [comment]
fixed via route2OD.py (#9117)

**Comment by behrisch** [comment]
just as a side note marouter does this aggregation as well

**Comment by namdre** [comment]
The main goal here was to support a workflow like this:
- obtain routes from some source (i.e. randomTrips)
- **transform this into an OD matrix with the help of a Taz file**
- edit the OD matrix (i.e. in netedit)
- rebuild traffic with od2trips

This should be easier than creating an OD matrix from scratch.
How could marouter help here?
